src/NuGet.Clients/NuGet.CommandLine/RegistryKeyUtility.cs

```diff
@@ -9,7 +9,7 @@ namespace NuGet.CommandLine
 {
     public static class RegistryKeyUtility
     {
-        public static object GetValueFromRegistryKey(string name, string registryKeyPath, RegistryKey registryKey, ILogger logger)
+        public static object GetValueFromRegistryKey(string name, RegistryKey registryKey, ILogger logger)
```

Comment:
Actually, `registryKeyPath` parameter is the registry key path open in `RegistryKey` object. Name is the 'name of the vaulue.

https://github.com/NuGet/NuGet.Client/blob/cc1e4ca108a97dc2af269ad5b80e94314c6b9f6c/src/NuGet.Clients/NuGet.CommandLine/RegistryKeyUtility.cs

We are more than happy to accept this contribution if you can change `name` to `registryKeyPath` in line 16 and add some unit tests.

Reply:
how would i add a unit test for an un-used parameter?

Reply:
@dominoFire 

Can that be handled separately? 
This PR focuses on removing the unused param only. 

@SimonCropp 

The build is failing with the following: 

```
##[error]src\NuGet.Clients\NuGet.CommandLine\Program.cs(31,30): Error CA1823: Unused field 'FilesystemRegistryKey'.
     8>C:\a\1\s\src\NuGet.Clients\NuGet.CommandLine\Program.cs(31,30): error CA1823: Unused field 'FilesystemRegistryKey'. [C:\a\1\s\src\NuGet.Clients\NuGet.CommandLine\NuGet.CommandLine.csproj]
##[error]src\NuGet.Clients\NuGet.CommandLine\Program.cs(30,30): Error CA1823: Unused field 'OSVersionRegistryKey'.
     8>C:\a\1\s\src\NuGet.Clients\NuGet.CommandLine\Program.cs(30,30): error CA1823: Unused field 'OSVersionRegistryKey'. [C:\a\1\s\src\NuGet.Clients\NuGet.CommandLine\NuGet.CommandLine.csproj]
##[error]src\NuGet.Clients\NuGet.CommandLine\Program.cs(32,30): Error CA1823: Unused field 'DotNetSetupRegistryKey'.
```

Can you please take a look at that?

Reply:
I created a new PR to use `registryKeyPath` parameter. More details at: https://github.com/NuGet/NuGet.Client/pull/4346